### DISCUSSION TITLE
Increase fabricator speed when docked at outposts

### DISCUSSION THREAD

**Discussion by Valentin-Metz** [FakeFishGames/Barotrauma#9976] [Ideas and suggestions]
I often find myself at an outpost with more than 5 minutes of pure crafting time ahead.
Even with skill 100 several items still take a rather long time to craft.
This doesn't add anything, as you are in perfect safety anyways.
(Below my (partial) crafting queue after completing 2 caves)

![image](https://user-images.githubusercontent.com/31850924/193544614-c801cd83-ec47-4b29-8539-9726dd075266.png)

My suggestion would be to either give a flat boost (at least 10x speed) to all fabricators when docked (similar to how fuel-rod consumption is cut), or if you want to motivate the player to leave their ship to only give outpost fabricators this speed boost.

**Comment by mygamingaccount** [comment]
Get used to a system where you keep ingredients in a fab and whoever walks past will click create

You don't need that much morphine anyway

**Reply by Valentin-Metz** [reply]
Not all subs come with all fabricators (I do not have a medical fabricator on board).
And I definitely need to craft all my materials away, if only for selling them at higher prices.

**Reply by mygamingaccount** [reply]
A Button clicker game Barotrauma is not. Complete missions if you want to make money.
The med fab side hustle will earn you a few hundred creds at best.

**Reply by Valentin-Metz** [reply]
This is not an edge case.
If you are in a solo campaign you will spend multiple *hours* sitting in front of fabricators and recyclers just waiting for them to complete. Even just recycling ores from one mining run can easily take up 5+ min.
You can not use the fabricator while "passing it by", because you are the only person on that ship.
Crafting 16 plastiseal takes 15\*8+5\*25 seconds.
That's 4 minutes.
You want your medic to carry at least that amount with more of it in storage.
This adds up massively.
Crafting a *single stack* of fentanyl takes over 6 minutes.
And this is all just for characters that meet the crafting skill requirements.
So if I want 4 stacks of fentanyl (which is a perfectly reasonable amount; crew of 8, each with auto injector headset, 8 more for the medic, 2 stacks in storage), I'll be forced to sit in front of the medical fabricator at the station with nothing else to do for over 25 minutes.
You can not even walk away and sort inventory because you need to click the button again every 20 seconds.

**Reply by mygamingaccount** [reply]
Here's a mod for you
https://steamcommunity.com/sharedfiles/filedetails/?id=2548565390

**Reply by Valentin-Metz** [reply]
The mod seems to be outdated and increases fabrication speed overall.
Fabrication speed limits make sense while out on a mission.
Desperately scrapping wrenches to get more railgun shells so you can finish the fight with an abyssal monster is fun.
Spending 12.5 minutes crafting 30 railgun shells in perfect safety to prepare yourself for the next one is not.

The amount of crafting time required for solo players in all aspects of the crafting system speaks for itself here I think.

**Reply by mygamingaccount** [reply]
You didn't even check that mod, it's A-OK.

Are you against practical solutions to your problem?
